core/service/app.go

```diff
@@ -62,6 +66,118 @@ func (a App) FindAppByID(ct context.Context, appID uint64) (entity.App, *errs.Er
 	return a.appDao.FindAppByID(ct, appID)
 }
 
+func (a App) FindSecretsByAppID(ct context.Context, appID uint64) ([]entity.AppSecret, *errs.Error) {
+	return a.appSecretDao.FindSecretsByAppID(ct, appID)
+}
+
+func (a App) CreateAppSecret(ct context.Context, appID uint64, name string) (entity.AppSecret, *errs.Error) {
```

Comment:
```suggestion
func (a App) CreateAppSecret(ct context.Context, appID uint64, input CreateAppSecretInput) (entity.AppSecret, *errs.Error) {
```

Define input type so that we can easily add more fields in the future without changing API signature.